MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/AddNewEntryManually.java

```diff
@@ -94,11 +107,22 @@ public void onDatePubSet(int datePub) {
     @Override
     public void onClick(View v) {
         if (v.getId() == R.id.btn_set_cover) {
-            mAddCover = addCover.getText().toString();
-            coverPreview.setVisibility(View.VISIBLE);
-            Picasso.with(this)
-                    .load(mAddCover)
-                    .into(coverPreview);
+//            Log.e("WTF", addCover.getText().toString());
```

Comment:
Do you want to keep the commented code? Usually it's not a good practice to keep commented code because the next developer which comes to the project will be a little bit confused why is that commented. If you do want to keep it then would be good to add at the top a comment why.